Tests/Editor/BuildPlayerTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.IO;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Tests
+{
+	[InitializeOnLoad]
+	public class BuildPlayerTest
+	{
+		const string kTestPath = "_Test";
+		const string kVerifySavingAssets = "VerifySavingAssets";
+
+		bool? m_VerifySavingAssets;
+
+		[SetUp]
+		public void SetUp()
+		{
+			if (EditorPrefs.HasKey(kVerifySavingAssets))
```

Comment:
Need better workflow for this. If I run tests one-by-one, then I get the prompt each time.